src/components/NavBar/Buttons.js

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Menu from "../Icons/Menu";
+import Search from "../Icons/Search";
+import User from "../Icons/User";
+import colors from "../../theme/colors";
+import spacing from "../../theme/spacing";
+import typography from "../../theme/typography";
+import { mediumAndUp, largeAndUp } from "../../theme/mediaQueries";
+import { Consumer } from "./Context";
+import getRelByTarget from "../../utils/link";
+
+const BaseButton = styled.button`
```

Comment:
Maybe this component should live in the Buttons directory? I think it could potentially be reused for building Tab components et. al.

Reply:
There is no tab component.

Reply:
I meant down the road